### ISSUE TITLE
provide a way to limit the data fetched?

### ISSUE THREAD

**Issue by fantasticsid** [jshemas/openGraphScraper#31]
If url points to a large file (link to a document for example), it downloads the whole file before parsing the content. Since we are using cheerio, we probably can provide a way to specify the maximum bytes fetched, and still be able to get open graph tags from partial content?

**Comment by jshemas** [comment]
@fantasticsid Do you have an example URL of this, so I can test it?

**Comment by fantasticsid** [comment]
https://downloads.slack-edge.com/mac_releases/Slack-2.3.2-macOS.zip

You can see the network request from activity monitor on mac, it will download the file. I think its just node request fetching the content

**Comment by jshemas** [comment]
So we use `Request` to to do the actual HTTP call. I just looked at their readMe and there doesn't seem to be a build in limit. I did how ever find a stack overflow post on this topic.

http://stackoverflow.com/questions/25080121/node-js-and-request-limiting-the-file-size-of-downloaded-file

I can probably add this in, in the near future. But if you want to submit a pull request, that would be awesome.
